src/views/Restoration/RestorationBusinessName.vue

```diff
@@ -6,105 +6,20 @@
-    <section class="mt-10" v-show="isTypeCoop">
-      <header id="association-type-header">
-        <h2>Cooperative Association Type</h2>
-        <!-- Help Section -->
-        <div class="mt-4">
-          <span class="help-btn" @click="coopHelpToggle = !coopHelpToggle">
-            <v-icon color="primary" style="padding-right: 5px">mdi-help-circle-outline</v-icon>
-            <span v-if="!coopHelpToggle">Help with Cooperative Association Types</span>
-            <span v-else>Hide Help</span>
-          </span>
-          <section v-show="coopHelpToggle" class="coop-type-help">
-            <header id="coop-type-help-header">
-              <h2>Help with Cooperative Association Types</h2>
-            </header>
-
-            <p><strong>Community Service Cooperatives</strong> are a particular kind of cooperative
-              recognized under the Cooperative Association Act. Community Service Cooperatives have a
-              similar status to that of non-profit societies. This type of cooperative also requires the
-              inclusion of non-alterable clauses in their rules to ensure they operate on a non-profit
-              basis, and their purpose is charitable, or to provide health, social, educational, or other
-              community services. Community Service Cooperatives cannot be Housing Cooperatives and cannot
-              issue investment shares.
-            </p>
-            <p><strong>Housing Cooperatives</strong> are a specific type of cooperative incorporated under
-              the Cooperative Association Act that provides housing to its members. Members may purchase
-              shares to join the Housing Cooperative and elect directors who will govern the cooperative.
-              Housing Cooperatives cannot issue investment shares. The Cooperative Association Act details
-              special provisions for Housing Cooperatives that need to be considered when deciding to
-              incorporate.
-            </p>
-            <p>An <strong>Ordinary Cooperative</strong> is a cooperative that may have a wide range of
-              purposes and is neither a Housing nor a Community Service Cooperative. The cooperative may
-              operate as a for-profit association and may issue investment shares.
-            </p>
-            <u class="help-btn" @click="coopHelpToggle = !coopHelpToggle"><small>Hide Help</small></u>
-          </section>
-        </div>
-      </header>
-
-      <div :class="{ 'invalid-section': getShowErrors && !hasValidCooperativeType }">
-        <v-card flat class="step-container">
-          <CooperativeType
-            :showErrors="getShowErrors"
-            @hasCooperativeType="onCooperativeType($event)"
-          />
-        </v-card>
-      </div>
-    </section>
-
-    <!-- Registered Office Addresses -->
-    <section class="mt-10" v-show="isEntityType">
+        <!--Restoration Type is here-->
+        <section class="mt-10" v-show="isEntityType">
```

Comment:
This v-show is suspicious.
1. I'm not sure why you need to check this here. Is entity type not set sometimes?
2. Why it is a v-show instead of a v-if? Do the component within need to be rendered when we don't know entity type?